akka-typed/src/test/scala/akka/typed/ActorContextSpec.scala

```diff
@@ -73,6 +73,36 @@ object ActorContextSpec {
   final case class GetAdapter(replyTo: ActorRef[Adapter], name: String = "") extends Command
   final case class Adapter(a: ActorRef[Command]) extends Event
 
+  val foo1: Actor.Stateless[Command] = Actor.Stateless {
+    case (_, Ping(replyTo)) ⇒
+      replyTo ! Pong2
+    case (_, Throw(ex)) ⇒
+      throw ex
+    case _ ⇒ ()
+  }
+
+  val foo2: Behavior[Command] = Actor.Stateless {
+    case (_, Ping(replyTo)) ⇒
+      replyTo ! Pong2
+    case (_, Throw(ex)) ⇒
+      throw ex
+    case _ ⇒ ()
+  }
+
+  val foo3: Behavior[Ping] = Actor.Stateless {
+    case (_, Ping(replyTo)) ⇒
+      replyTo ! Pong2
+  }
+
+  // TODO missing parameter type for expanded function The argument types of an anonymous function must be fully known.
+  //      (SLS 8.5) Expected type was: Function2[akka.typed.scaladsl.ActorContext[akka.typed.ActorContextSpec.Ping], akka.typed.ActorContextSpec.Ping, ?]
+  /*
+  val foo4 = Actor.Stateless[Ping] {
```

Comment:
Wow, that’s a funny one! Changing the expected function’s return type from wildcard to Any makes it work.

Although I am pretty sure that we should not document this pattern: it is unsafe (actor dies from unknown messages) and wastes object allocations.